### PR TITLE
[Emotion v10] Finish upgrading all things to emotion 10

### PR DESCRIPTION
**Pull request by snags88**
This PR adds the following to fully convert to emotion 10
- Use `babel-preset-css-prop`
- Remove `babel-plugin-emotion`
- Update all `css` prop to not use string and align with new usage from emotion 10.
  - This includes docs, stories, and src files.
  - Also includes the svg -> react component template that SVGR uses.
- Update jest configuration to use new `jest-emotion` config in serializer.
- Update babel configurations to stop using emotion plugin and use the emotion css prop preset.
- Update rollup config globals.

### CODE REVIEW

File: src/shared-components/alert/index.js [hunk 1]
```diff
@@ -82,7 +83,7 @@ class Alert extends React.Component {
       >
         <AlertContentContainer>
           <Icon
-            css={[alertIconStyles]}
+            css={css`${alertIconStyles};`}
```
Comment (ZeMunchkin) [inline src/shared-components/alert/index.js]: Can you clarify something? `alertIconStyles` is created in style.js using `css`. Is there a difference between creating it with `css` vs wrapping in `css` here?

File: src/shared-components/selectorButton/index.js [hunk 2]
```diff
@@ -45,9 +46,9 @@ const SelectorButton = ({
   >
     <SelectorContainer>
       <CheckmarkIcon
-        css={`
-          color: ${checked ? COLORS.white : 'transparent'};
+        css={css`
           ${selectorIcon};
+          color: ${checked ? COLORS.white : 'transparent'};
```
Comment (ZeMunchkin) [inline src/shared-components/selectorButton/index.js]: 🙌 Good refactor on ordering

File: src/shared-components/alert/index.js (hunk 1 above)
Comment (snags88) [reply src/shared-components/alert/index.js]: oh nope, there's not. Both ways here are valid. https://emotion.sh/docs/composition

I just did a global search on `css=` and updated all of it to use the template literal.

Comment (ZeMunchkin) [reply src/shared-components/alert/index.js]: 👍

**Pull request merged**